packages/core/src/styles/Button.scss

```diff
@@ -121,3 +121,18 @@
         opacity: .5;
     }
 }
+
+// ----------------------
+//  Custom <Button> tag
+// ----------------------
+button.#{$prefix}-button {
+    background: transparent;
+    border: 0;
+    outline: 0;
+    appearance: none;
+    user-select: none;
+}
+
+a.#{$prefix}-button {
+    text-decoration: none;
```

Comment:
Do we need to eliminate the color change effect on hover and on visited as well?

```
a, a:visited, a:hover, a:active {
  color: inherit;
}
```

Reply:
The default link colors are already overridden by color-variant modifiers.
Specifying colors here will eventually override those modifiers.